### PR TITLE
Add instructions for running Maintainer meetings

### PR DESCRIPTION
**Pull request by angela-li**

### CODE REVIEW

File: topic_folders/maintainers/maintainers.md [hunk 1]
```diff
@@ -91,4 +91,44 @@ the onboarding process.
...
 [DC GH Lesson Maintainer Teams]: https://github.com/orgs/datacarpentry/teams/lesson-maintainers
 [LC GH Lesson Maintainer Teams]: https://github.com/orgs/librarycarpentry/teams/lesson-maintainers
-[The Carpentries GH Lesson Maintainer Teams]: https://github.com/orgs/carpentries/teams/lesson-maintainers
\ No newline at end of file
+[The Carpentries GH Lesson Maintainer Teams]: https://github.com/orgs/carpentries/teams/lesson-maintainers
+
+
+## Maintainer Community Meetings
+
+The Maintainer community meets on the third Wednesday of every month. Times for these meetings can be found on the [Community Calendar](https://static.carpentries.org/community/#community-events). A reminder for these meetings should be sent out to the Maintainers email list one week before the meeting takes place, and the [Maintainers Etherpad](https://pad.carpentries.org/maintainers) should also be updated. Ideally, these meetings will be co-facilitated by two members of the Carpentries team.
```
Comment (ErinBecker) [inline topic_folders/maintainers/maintainers.md:99]: Could we change "two members of the Carpentries team" to "a member of [The Carpentries Core Team](https://carpentries.org/team/) and the Maintainer Community Development Lead"?

File: topic_folders/maintainers/maintainers.md [hunk 2]
```diff
@@ -91,4 +91,44 @@ the onboarding process.
...
+
+
+```
+Join from PC, Mac, Linux, iOS or Android: {Zoom link} 
+
+Meeting 1: Wednesday, {Date} at 16:00 UTC
+Please click this link to convert to your local time: https://www.timeanddate.com/worldclock/fixedtime.html?iso=20200617T16 (edit this)
+
+Meeting 2: Wednesday, {Date} at 21:00 UTC
+Please click this link to convert to your local time: https://www.timeanddate.com/worldclock/fixedtime.html?iso=20200617T21 (edit this)
```
Comment (ErinBecker) [inline topic_folders/maintainers/maintainers.md:111]: I recommend taking out the time and date link and leaving a placeholder like (add time converter link using [TimeandDate's Event Time Announcer](https://www.timeanddate.com/worldclock/fixedform.html)).

File: topic_folders/maintainers/maintainers.md [hunk 3]
```diff
@@ -91,4 +91,44 @@ the onboarding process.
...
+- {name}
+- {name}
+
+Timekeeper:
+Notetaker (please add notes within agenda below):
+
+Agenda:
+
+- Introductions (5 min)
+- Updates from Carpentries team (15 min):
```
Comment (ErinBecker) [inline topic_folders/maintainers/maintainers.md:129]: Add placeholder agenda item for [current RFCs](https://github.com/carpentries/maintainer-RFCs/issues)

**Comment by maneesha** [comment]
Thanks @angela-li  and @ErinBecker !

**Pull request merged**